.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -0,0 +1,169 @@
+**Changes:**
+
+* IoT3 SDK:
+    * core: ...
+    * mobility: ...
+* applications:
+    * iqm: ...
+    * ...
+
+Closes: #
+
+---
+**How to test:**
+
+_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
+
+1. Prepare a test environment:
+    1. be sure to have unrestricted access to _test.mosquitto.org_ (IPv4 and IPv6)
+    2. be sure to have an MQTT broker that listens locally on port 1883, with no credentials and no ACL; if not, run your own:
+        ```sh
+        $ mosquitto
```

Comment:
I would prefer a command not starting with '#', to be able to use the "copy" graphical function, or easily copy/paste the command. Like that, we must paster->click->go first digit->remove the first digit to execute. Moreover , "sh" is saying that it is a shell part. And moreover...I've got a #, not a $ at the beginning of my terminal :D.

Reply:
NACK. Using the sharp `#` or `$` signs is a well-established way to differentiate commands tht should be used as root (`#`) or as a standard user (`$`).

Reply:
okay, but what about...nothing?

Reply:
```sh
mosquitto
```